firefox-ios/Client/Frontend/Library/Reader/ReaderPanel.swift

```diff
@@ -26,7 +26,7 @@ private struct ReadingListTableViewCellUX {
     static let MarkAsReadButtonTitleEdgeInsets = UIEdgeInsets(top: 4, left: 4, bottom: 4, right: 4)
 }
 
-private struct ReadingListPanelUX {
+struct ReadingListPanelUX {
```

Comment:
thanks for cleaning this up! What do you think about moving this struct to the EmptyStateView and keeping it private so its self contained in that file?

Reply:
Sure, I completely agree. That makes sense and will keep the code more self-contained, reducing the risk of misuse. I'll make the changes.